geonode_mapstore_client/client/static/mapstore/configs/localConfig.json

```diff
@@ -564,7 +564,7 @@
                         },
                         {
                             "type": "link",
-                            "href": "/apps/new?appType=GeoStory",
+                            "href": "/viewer/#/geostory/new",
```

Comment:
there is another href `/apps/new?appType=GeoStory` inside the localConfig.json, please that one also

Reply:
Thanks Dont know how i missed this